src/eeeoh/eeeoh.ts

```diff
@@ -713,16 +721,34 @@ export const createOptions = <CustomLevels extends string>(
     mixin: (mergeObject, level, logger) => {
       const tier = getTier(mergeObject, level, logger);
 
+      let ddTags = {};
+
+      if (opts.eeeoh && base?.env && base.version) {
+        const config = getConfigForLogger(logger);
+        const { team } = config ?? opts.eeeoh;
+        const { env, version } = base;
+
+        ddTags = {
+          ddtags: ddtags({
+            env,
+            version,
+            ...(team ? { team } : {}),
+          }),
+        };
+      }
```

Comment:
Do you see much opportunity in consolidating this logic into `formatOutput`? Maybe there could be a `getTier` equivalent like `getTags` for consistency.

Reply:
Ya good call I've had a crack.